addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -63,7 +63,7 @@ private _oldCompats = [];
 } forEach _addons;
 
 if (_oldAddons isNotEqualTo []) then {
-    _oldAddons = _oldAddons apply {"%1.pbo", _x};
+    _oldAddons = _oldAddons apply { format["%1.pbo", _x] };
```

Comment:
```suggestion
    _oldAddons = _oldAddons apply { format ["%1.pbo", _x] };
```

Reply:
I'd actually like to run a find and replace against stuff like this, may be a minor pain on some PRs but imo worth it